check-pipe totally broken

Since 3.1.2 check-pipe is totally broken.

```
<div *ngIf="countingPoints$ | async as countingPoints; else countingPoinsAreLoading">
  {{countingPoints}}
</div>
<ng-template #countingPoinsAreLoading>loading</ng-template>
```

> ERROR: XXX.component.html[2, 13]: The pipe operator should be surrounded by one space on each side, i.e. " | ".

@mgechev, sorry, but I can't see how this error is related to this issue. Can you explain?

I searched a little more, looks like there are other issues more likely to math the errors above: #190 and angular/angular-cli/issues/6554

This issue appeared after I updated from `3.0.1` to `3.1.0` or `3.0.2`:
```
~/proj/jexia-webapp [work ~2 *] » npm i -D codelyzer@3.1.0                              willgm@localhost
+ codelyzer@3.1.0
updated 1 package in 4.362s
---------------------------------------------------------------------
~/proj/jexia-webapp [work ~2 *] » npm run lint                                          willgm@localhost

> web-app@0.0.3-alpha lint /home/willgm/proj/jexia-webapp
> ng lint --type-check


ERROR: /home/willgm/proj/jexia-webapp/src/app/project/project-features/project-features.component.html[1, 2]: The property "async" that you're trying to access does not exist in the class declaration.
ERROR: /home/willgm/proj/jexia-webapp/src/app/project/project-features/project-features.component.html[55, 34]: The property "async" that you're trying to access does not exist in the class declaration.

Lint errors found in the listed files.
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! web-app@0.0.3-alpha lint: `ng lint --type-check`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the web-app@0.0.3-alpha lint script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/willgm/.npm/_logs/2017-07-05T18_17_21_680Z-debug.log
---------------------------------------------------------------------
~/proj/jexia-webapp [work ~2 *] » npm i -D codelyzer@3.0.1                              willgm@localhost
+ codelyzer@3.0.1
updated 1 package in 4.294s
---------------------------------------------------------------------
~/proj/jexia-webapp [work ~2 *] » npm run lint                                          willgm@localhost

> web-app@0.0.3-alpha lint /home/willgm/proj/jexia-webapp
> ng lint --type-check



All files pass linting.
```

--- a/test/angularWhitespaceRule.spec.ts
+++ b/test/angularWhitespaceRule.spec.ts
@@ -204,7 +204,25 @@ describe('angular-whitespace', () => {
         `;
         assertSuccess('angular-whitespace', source, ['check-pipe']);
       });
+
+      it('should succeed with *ngIf and else condition', () => {
+        let source = `
+        @Component({
+          selector: 'foo',
+          template: \`
+            <div *ngIf="countingPoints$ | async as countingPoints; else countingPoinsAreLoading">
+              {{countingPoints}}
+            </div>
+            <ng-template #countingPoinsAreLoading>loading</ng-template>
+          \`
+        })
+        class Bar {}
+        `;
+        assertSuccess('angular-whitespace', source, ['check-pipe']);
+      });
+
     });
+
 
     describe('check-semicolon', () => {
 

